resolwe_bio/tests/processes/test_expression.py

```diff
@@ -18,13 +18,13 @@ def test_cufflinks(self):
             annotation_gtf = self.prepare_annotation('annotation dicty.gff.gz')
             annotation_gff3 = self.prepare_annotation_gff()
 
-            inputs = {
+            aligned_reads = self.run_process('alignment-hisat2', {
                 'genome': genome.pk,
                 'reads': reads.pk,
-                'annotation': annotation_gtf.pk,
-                'PE_options': {
-                    'library_type': "fr-unstranded"}}
-            aligned_reads = self.run_process('alignment-tophat2', inputs)
+                'spliced_alignments': {
```

Comment:
How is this change connected to the process removal?

Reply:
`alignment-tophat2` was removed as a process and was used in a few of the tests. `alignment-hisat2` is more modern replacement aligner to be used in this workflows, so this replacement was made.